web/src/stream_create.js

```diff
@@ -435,16 +457,19 @@ export function set_up_handlers() {
         const name_ok = stream_name_error.validate_for_submit(stream_name);
 
         if (!name_ok) {
+            stream_settings_components.show_subs_pane.create_stream("configure_channel_settings");
             return;
         }
 
         const principals = stream_create_subscribers.get_principals();
         if (principals.length === 0) {
             stream_subscription_error.report_no_subs_to_stream();
+            stream_settings_components.show_subs_pane.create_stream("subscribers_container");
```

Comment:
Is this needed, since user will already be on subscribers panel.

The code for `name_ok` part will still be needed to handle the case where some other user creates a stream with same name in the time this user is adding subscribers.